draft-js-dnd-plugin/src/modifiers/onDropBlock.js

```diff
@@ -0,0 +1,40 @@
+import AddBlock from './addBlock';
+import RemoveBlock from './removeBlock';
+import { Entity } from 'draft-js';
+
+export default function onDropBlock() {
+  return function onDropBlockInner(e) {
+    const { selection, dataTransfer, editorState, onChange } = e;
+
+    const state = editorState();
+
+    // Get data 'text' (anything else won't move the cursor) and expecting kind of data (text/key)
+    const raw = dataTransfer.data.getData('text');
+    const data = raw ? raw.split(':') : [];
```

Comment:
Yeah, this is something I wanted to give another thought. If you start dragging a draggable block, it will add a text key into dataTransfer that looks like this: `key:${block.key}`. The problem with dataTransfer is, you cannot add json (except if its stringified, which would be a pain to use, as you'd have to try parse the stringified json, it could still only be some text dropped on the editor).

Maybe I will change the `key:${block.key}` to something more unique like `DRAFTJS_DND_KEY:${block.key}` to really avoid conflicts. We could also add the blocks key to `window.CURRENT_DRAFTJS_DND_KEY` and read it from there, but I think everyone will agree this isn't a wise solution :)

The editor will also take care of `type:${some_block_type}` and create a block accordingly. This could be renamed to `DRAFTJS_BLOCK_TYPE:${some_block_type}`.

Maybe a good opportunity for string constants, gonna work on that.